.github/workflows/ci-disable-gtest.yml

```diff
@@ -37,12 +37,12 @@ jobs:
         toolchain: stable
         override: true
 
-    - name: Cache external dependencies
-      id: cache-ext
-      uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 # v3.3.1
-      with:
-        path: ext
-        key: ${{ runner.os }}-disable-gtest-${{ hashFiles('ext/*.cmd') }}
+    - name: Install cargo-c
```

Comment:
I've set up LocalRav1e to build the cargo-c crate as part of the cmake project  only if cargo-c isn't already available. It takes longer to build cargo-c than all of the other dependencies combined, so it seemed like a sensible optimization to use the binaries provided by the project releases. Particularly since the question of how to cache the external dependency builds in CI with FetchContent/ExternalProject is unresolved.

Reply:
I was trying to keep the size of this PR down, but I think ideally it should be an option to FetchContent the pre-built binaries provided by cargo-c, and even rav1e, configurable with options.